chem/miller.py

```diff
@@ -18,9 +18,10 @@ def lcm(a, b):
         a, b: floats
 
     Returns:
-        float
+        int: Because math.gcd will only return int after python 3.9
+        Previously it depended on type of args, now it'll only return int
     """
-    return a * b / fr.gcd(a, b)
+    return a * b / math.gcd(int(a), int(b))
```

Comment:
I am not sure if this is the right way to do it, can anyone else take a look into this ? 

Had to change from `fr.gcd` to `math.gcd` because since python 3.9 it Fractions no longer support gcd and it has been moved to `math`  [https://docs.python.org/3/library/fractions.html] (ref)